source/guides/maintenance/container-image-modify.rst

```diff
@@ -93,131 +102,103 @@ Dockerfile.
 
 This example shows how to rebuild single-stage containers against a specific
-OS version, :file:`<CL_VERSION>`, by adding a new argument to the Docker build command line.
-
-#. Open a Terminal on |CL| and enter the
-   :file:`dockerfiles/machine-learning-ui` directory.
+OS version, :file:`<CL_VERSION>`, by adding a new argument to the Docker build
+command line. 
 
 #. Rebuild the :file:`clearlinux/machine-learning-ui`. Add an extra build
-   argument :command:`swupd_args="-m <CL_VERSION>"`; in this case, the build version is 31090.
+   argument :command:`swupd_args="-m <CL_VERSION>"`; in this case, the build
+   version is 31110.
 
    .. code-block:: bash
       :linenos:
-      :emphasize-lines: 6
+      :emphasize-lines: 5
 
       docker build \
       --no-cache \
       --build-arg http_proxy=$http_proxy \
       --build-arg https_proxy=$https_proxy \
-      -t clearlinux/machine-learning-ui:31090 \
-      --build-arg swupd_args="-m 31090" .
+      --build-arg swupd_args="-m 31110" \      
+      --tag clearlinux/machine-learning-ui:31110 \
+      machine-learning-ui/
 
 #. Run the docker container image:
 
    .. code-block:: bash
 
-      docker run clearlinux/machine-learning-ui:31090 swupd info
+      docker run clearlinux/machine-learning-ui:31110 swupd info
 
 #. Sample output shows:
 
    .. code-block:: console
 
       Distribution:      Clear Linux OS
-      Installed version: 31090
+      Installed version: 31110
       Version URL:       https://cdn.download.clearlinux.org/update
       Content URL:       https://cdn.download.clearlinux.org/update
 
-Example 3: Change Clear Linux OS version (multi-stage Dockerfiles)
-===================================================================
 
-This example shows how to rebuild multi-stage Dockerfiles to use a specific
-OS version.
+Example 3: Change |CL| version (multi-stage build)
+**************************************************
+
+This example shows how to rebuild the cgit Dockerfile to use a specific |CL|
+version. The clearlinux/cgit Dockerfile has a multi-stage build with multiple
+layers: *os-core*, *httpd*, and *cgit*. This can be used as reference for
+building other multi-stage images with any number of layers. 
+
 
 .. important::
 
-   All upper layers of multi-stage Dockerfiles inherit the
-   |CL| version from the base layer. Rebuild the all underlying base layers against the desired OS version. In this example, four base layers must be rebuilt.
+   All upper layers of multi-stage Dockerfiles inherit the |CL| version from
+   the base layer. Rebuild the all underlying base layers against the desired
+   OS version. In this example, four base layers must be rebuilt.
 
-#. Change directory into :file:`dockerfiles`.
 
-First layer
------------
+First layer: os-core
+--------------------
 
 #. Rebuild the first layer, *os-core*. Add an extra build argument
    :command:`swupd_args="-m <CL_VERSION>"`; in this case, the build
-   version is 31090.
+   version is 31110.
```

Comment:
Example 3, First layer, Step 1: 
Is it possible you could follow the same pattern in this example as you did in Example 1, Step 2-3? (Ln 50).

Reply:
Thanks for clarifying, @puneetse. Confusion caused by a lack of GH to render code-blocks with emphasize lines.